### ISSUE TITLE
If selected is empty all data will become Nan

### ISSUE THREAD

**Issue by Joshuaoneheart** [subhadarship/kmeans_pytorch#41]
You don't handle the case when non of data is clustered into a cluster and if the num of cluster is big enough, all data will become nan.

**Comment by XA23i** [comment]
yes, this will cause deading loops and I think we should manually initialize that value randomly.
